operations/resourcecategories.md

```diff
@@ -0,0 +1,129 @@
+            "ServiceId": "9b3a6c54-63aa-4383-b50e-b0030078184b",
+            "IsActive": true,
+            "Type": "Bed",
+            "Names": {
+                "en-US": "Bed 1"
+            },
+            "ShortNames": {
+                "en-US": "Bed 1"
+            },
+            "Descriptions": {},
+            "Ordering": 2,
+            "Capacity": 1,
+            "ExtraCapacity": 0,
+            "ExternalIdentifier": null
+        },
+        {
+            "Id": "47d6b462-35ec-467e-a565-b00300781a41",
+            "EnterpriseId": "6b78e729-6271-4765-97f7-b0030078151d",
+            "ServiceId": "9b3a6c54-63aa-4383-b50e-b0030078184b",
+            "IsActive": true,
+            "Type": "Dorm",
+            "Names": {
+                "en-US": "Dorm 1"
+            },
+            "ShortNames": {
+                "en-US": "Dorm 1"
+            },
+            "Descriptions": {},
+            "Ordering": 1,
+            "Capacity": 4,
+            "ExtraCapacity": 0,
+            "ExternalIdentifier": null
+        }
+    ],
+    "Cursor": "47d6b462-35ec-467e-a565-b00300781a41"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ResourceCategories` | array of [Resource category](#resource-category) | optional | Resource categories of the resources. |
+| `Cursor` | string | required | Unique identifier of the last and hence oldest resource category returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older resource categories. |
+
+#### Resource category
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the category. |
+| `EnterpriseId` | string | required | Unique identifier of the [Enterprise](enterprises.md#enterprise). |
+| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) of the resource category. |
+| `IsActive` | bool | required | Whether the resource category is still active. |
+| `Type` | string | required | Type of the category. |
+| `Names` | [Localized text](#localized-text) | required | All translations of the name. |
+| `ShortNames` | [Localized text](#localized-text) | required | All translations of the short name. |
+| `Descriptions` | [Localized text](#localized-text) | required | All translations of the description. |
+| `Ordering` | number | required | Ordering of the category, lower number corresponds to lower category \(note that uniqueness nor continuous sequence is guaranteed\). |
+| `Capacity` | number | required | Capacity that can be served \(e.g. bed count\). |
+| `ExtraCapacity` | number | required | Extra capacity that can be served \(e.g. extra bed count\). |
+| `ExternalIdentifier` | string | optional, max 255 characters | Identifier of the resource category from external system. |
+
+#### Resource category type
+
+* `Room`,
```

Comment:
No need to add commas after each bullet point, we haven't done that anywhere else in the docs.